### PR TITLE
Remove ops from Layer Creator/ Node Converter - part 6

### PR DESCRIPTION
**Pull request by blesniewski**
Num | Removed op name
-- | --
1 | DeformableConvolution
2 | DeformablePSROIPooling
3 | MaxPool
4 | NonMaxSuppression
5 | GroupConvolutionBackpropData
6 | GroupConvolution

43473

### CODE REVIEW

**Comment by blesniewski** [comment]
Should be merged after [PR_3493](https://github.com/openvinotoolkit/openvino/pull/3493) is merged.

**Comment by ilyachur** [comment]
Please resolve merge conflicts.

**Comment by blesniewski** [comment]
@openvinotoolkit/openvino-ngraph-maintainers Seems like this PR can be merged.

File: inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp [hunk 1]
```diff
@@ -1292,6 +1292,46 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
...
+        return res;
+    });
+
+    addSpecificCreator({"DeformablePSROIPooling"}, [](const std::shared_ptr<::ngraph::Node> &node,
+                                                      const std::map<std::string, std::string> &params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PSROIPooling", details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<InferenceEngine::CNNLayer>(attrs);
+        res->params = params;
+        res->params["no_trans"] = node->get_input_size() == 2 ? "1" : "0";
+        // temporary workaround due to incorrect usage of group_size in the nGraph operation for the DeformablePSROIPooling
```
[the remark this answers is not in the document]
Comment (blesniewski) [reply inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp:1329]: yes

**Review by sdurawa — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**